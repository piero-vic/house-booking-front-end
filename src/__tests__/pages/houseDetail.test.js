import renderer from 'react-test-renderer';
import { BrowserRouter as Router } from 'react-router-dom';
import { Provider } from 'react-redux';
import store from '../../redux/store';
import HouseDetail from '../../components/HouseDetails';

it('renders correctly', () => {
  const tree = renderer
    .create(
      <Provider store={store}>
        <Router>
          <HouseDetail />
        </Router>
      </Provider>,
    )
    .toJSON();
  expect(tree).toMatchSnapshot();
});
