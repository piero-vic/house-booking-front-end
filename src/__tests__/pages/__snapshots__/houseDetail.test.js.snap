// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="w-11/12 mx-auto pt-20 gap-6 sm:flex"
>
  <img
    alt="house"
    className="h-4/5 w-4/5 mb-5 sm:min-w-0 min-w-full"
  />
  <div
    className="items-start"
  >
    <div
      className="text-right mb-4 text-gray-800"
    >
      <div
        className="font-semibold text-2xl tracking-tighter"
      />
      <div
        className="text-xs font-extrabold"
      />
    </div>
    <div
      className="text-center sm:text-right"
    >
      <a
        href="/reserve/undefined"
        onClick={[Function]}
      >
        <button
          className="text-sm font-semibold px-10 py-3 sm:mt-10 mt-5 leading-5 tracking-wide bg-lime-500 text-white rounded-full"
          type="button"
        >
          Reserve
        </button>
      </a>
    </div>
  </div>
</div>
`;
